Tests/Cosmos.System.Tests/DiskManagerTest.cs

```diff
@@ -36,24 +56,27 @@ public void CleanUp()
         }
 
         [Test]
-        public void TestDiskManagerQuickFormat()
+        public void Execute()
         {
-            string driveName = @"C:\";
-            var drive = new DiskManager(driveName);
-            drive.Format("FAT32", aQuick: true);
+            string driveName = @"0:\";
+
+            Assert.IsTrue(ourPart.RootPath == driveName, "ManagedPartition.RootPath failed drive has wrong name");
```

Comment:
Nitpick: Use Assert.AreEqual not IsTrue in future, it provides more helpful error messages when tests fail.